packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1095,6 +1095,941 @@ curl -v -X DELETE \
+        },
+        "self": {
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            },
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/welcome/customizations/oel2kk2VDW0K4AOZp0g4"
+        },
+        "template": {
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            },
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/welcome"
+        },
+        "test": {
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            },
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/welcome/test"
+        }
+    }
+}
+```
+
+### Update email customization
+
+<ApiOperation method="put" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations/${customizationId}" />
+
+Updates an email customization
+
+#### Request path parameters
+
+| Parameter         | Type        | Description                  |
+| ----------------- | ----------- | ---------------------------- |
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+| `customizationId` | String      | ID of an Email Customization |
+
+#### Request body
+
+The [Email Customization](#email-customization) resource to update.
+
+#### Response body
+
+The [Email Customization](#email-customization) resource that was updated.
+
+If `isDefault` is `true`, the previous default email customization will have its `isDefault` set to `false`.
+
+Returns a `409 Conflict` if:
+- there is already another email customization for the specified `language`.
+- `isDefault` is `false` and the email customization being updated is the default.
+
+Returns a `400 Bad Request` if:
+- `language` is not one of the [supported locales](#supported-locales).
+- `body` does not contain a required variable references.
+
+Passing an invalid `brandId`, `templateName`, or `customizationId` returns a `404 Not Found` with error code `E0000007`.
+
+> **Note:** See [email customization constraints](#email-customization-constraints) for more information.
+
+#### Use examples
+
+The following example updates a customization for the `welcome` email template.
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "language": "en",
+  "subject": "Hello from Okta!",
+  "body": "<!DOCTYPE html><html>...${activationToken}...</html>",
+  "isDefault": true
+}' \
+'https://${yourOktaDomain}/api/v1/brands/${brandId}/templates/email/welcome/customizations/${customizationId}'
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+```
+
+```json
+{
+		"body": "<!DOCTYPE html><html>...${activationToken}...</html>",
```

Comment:
too much spacing here as well :)

Reply:
Thanks. Yeah I noticed these after I did the Netlify deploy. Not sure how I managed to do that. :)